documentation/examples/prometheus-kubernetes.yml

```diff
@@ -60,6 +61,7 @@ scrape_configs:
   - api_servers:
     - 'https://kubernetes.default.svc'
     in_cluster: true
+    role: endpoint
 
   relabel_configs:
   - source_labels: [__meta_kubernetes_role, __meta_kubernetes_service_annotation_prometheus_io_scrape]
```

Comment:
I see potential problem for the job kubernetes-cluster as there is a combination of two roles. If you set role to apiserver directly, you will not have entries for nodes targets.